src/jit/compatjit/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+project(compatjit)
+
+# This compatjit.dll is only built if we are not building JIT32 as compatjit.dll.
```

Comment:
Does this in the open build legacy_backend twice? Giving two dlls, compatjit.dll and legacyjit.dll which are the same thing?